<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>xgConstraintChecker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/plugins/xgConstraintChecker/checker_config.png</normaloff>:/plugins/xgConstraintChecker/checker_config.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Constraint Check to run:</string>
   </property>
   <widget class="QCheckBox" name="chk_word_report">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Produce Word report</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grp_report">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Report Details:</string>
    </property>
    <widget class="QLabel" name="lbl_report_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Save report as:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_created_by">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Created by:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_word_report">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_browse">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_created_by">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="lst_checks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_word_report</sender>
   <signal>toggled(bool)</signal>
   <receiver>grp_report</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_browse</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lst_checks</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>runSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileDialog()</slot>
  <slot>runSelected(QListWidgetItem*)</slot>
 </slots>
</ui>
